.drone.yml

```diff
@@ -84,6 +84,25 @@ pipeline:
       local: false
       event: [ push ]
 
+  update-search-index:
+    image: algolia/docsearch-scraper
+    entrypoint: ""
+    pull: true
+    secrets: [ 
+      algolia_app_id, algolia_config, 
+      algolia_index_name, algolia_search_api_key, 
+      algolia_write_api_key 
+    ]
+    environment:
+      - ALGOLIA_APP_ID=${ALGOLIA_APP_ID}
+      - ALGOLIA_API_KEY=${ALGOLIA_SEARCH_API_KEY}
+      - APPLICATION_ID=${ALGOLIA_APP_ID}
+      - API_KEY=${ALGOLIA_WRITE_API_KEY}
+      - ALGOLIA_INDEX_NAME=${ALGOLIA_INDEX_NAME}
+      - CONFIG=${ALGOLIA_CONFIG}
+    commands:
+      - pipenv run python -m src.index
```

Comment:
Why does it need this pipenv stuff? Maybe we got to build our own Docker image for that...

Reply:
It seemed to be what was required, as I understood it at the time.

Reply:
From re-reading https://community.algolia.com/docsearch/run-your-own.html#running-your-config, the pipeline only needs the docsearch tool available, a Docker image isn't required, along with the config file.

Reply:
Just made a further update to .drone.yml. How close is it now?